server/services/prismic.js

```diff
@@ -129,6 +131,42 @@ export async function getArticleList(page = 1, {pageSize = 10, predicates = []}
   }: PaginatedResults);
 }
 
+export async function getArticleSeries(seriesId) {
+  const prismic = await prismicApi();
+
+  const articlesSchedule = await prismic.getByID(seriesId).then(s =>  {
+    const schedule = s.data.schedule.map(a => {
+      return Object.assign({}, a, {title: asText(a.title)});
+    });
+    return Object.assign({}, s.data, {schedule});
+  });
+
+  const publishedFromSeries = await prismic.query([
+    Prismic.Predicates.at('my.articles.series.series', seriesId)
+  ]);
+
+  const scheduleItems = articlesSchedule.schedule.map(articleInSchedule => {
+    const matchingArticle = publishedFromSeries.results.find(publishedArticle => {
+      return asText(publishedArticle.data.title) === articleInSchedule.title;
+    });
+
+    return {
+      url: matchingArticle ? `/articles/${matchingArticle.id}` : null,
+      contentType: matchingArticle ? getContentTypeFromPrismicType(matchingArticle.type) : null,
```

Comment:
Is this not always going to be `article`?

Reply:
Due to this:
https://github.com/wellcometrust/wellcomecollection.org/pull/1710/files?diff=split#diff-b6ea7a71e387114552e5d2ab051017dfR145

Reply:
yep, you're right